utilities/logic/async_wrapper.py

```diff
@@ -46,24 +46,26 @@ def daemonize_self():
         if pid > 0:
             # exit first parent
             sys.exit(0)
-    except OSError, e:
-        print >>sys.stderr, "fork #1 failed: %d (%s)" % (e.errno, e.strerror)
-        sys.exit(1)
+    except OSError:
+        e         = get_exception()
+        error_str = 'fork #1 failed: %d (%s)' % (e.errno, e.strerror)
+        module.fail_json(error_str)
```

Comment:
We don't have a module object to use here.  I'm also not certain if this portion of async_wrapper should be printing json.  I think at this point, the job is running in the background after forking from the main process.

(async is for taking a long running job and running it asynchronously, being able to come back to it later in the playbook and wait for it to have completed.  async_wrapper is used to start up the asynchronous job on the remote machine.)